vendor/acutest.h

```diff
@@ -1445,11 +1445,13 @@ test_cmdline_callback__(int id, const char* arg)
             test_worker_index__ = atoi(arg);
             break;
         case 'x':
+#if defined ACUTEST_UNIX__
             test_xml_output__ = fopen(arg, "w");
             if (!test_xml_output__) {
                 fprintf(stderr, "Unable to open '%s': %s\n", arg, strerror(errno));
                 exit(2);
             }
+#endif
```

Comment:
The XML output isn't used currently, so it is not overly important. However, is it necessary to exclude the entire case and not only the error logging?
```suggestion
            test_xml_output__ = fopen(arg, "w");
            if (!test_xml_output__) {
#if defined ACUTEST_UNIX__
                fprintf(stderr, "Unable to open '%s': %s\n", arg, strerror(errno));
#else
                fprintf(stderr, "Unable to open '%s'\n", arg);
#endif
                exit(2);
            }
```